charts/kubewarden-controller/Chart.yaml

```diff
@@ -14,6 +14,3 @@ maintainers:
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
 version: 1.1.1
-
-# This is the version of kubewarden-controller container image to be used
-appVersion: "v1.1.0"
```

Comment:
should this be kept around?
(Tangential, but note that for Rancher airgap, we can't depend on using appVersion on templating, as reading `tag` values happens before the chart has been templated)

Reply:
I've made a fixup commit and rebased, together with other things. Now it is kept around.